tests/temp_tests/test_launchable.py

```diff
@@ -53,10 +53,10 @@ def test_launchable_init():
 def test_invalid_job_name(wlmutils):
     entity = Application(
         "test_name",
-        run_settings="RunSettings",
         exe="echo",
         exe_args=["spam", "eggs"],
     )  # Mock RunSettings
+
```

Comment:
Nit: feel free to drop the (looks like) no longer needed comment
```suggestion
    )

```

Reply:
removed